metis-authentication/metis-authentication-rest/src/main/java/eu/europeana/metis/authentication/rest/AuthenticationController.java

```diff
@@ -147,7 +153,7 @@ public void deleteUser(@RequestHeader("Authorization") String authorization,
       throw new UserUnauthorizedException("Action allowed only from admin users");
     }
     authenticationService.deleteUser(userEmailToDelete);
-    LOGGER.info("User with email: {} deleted", userEmailToDelete);
+    LOGGER.info("User with email: {} deleted", StringEscapeUtils.escapeJava(userEmailToDelete));
```

Comment:
My eclipse complains that the expression is not done conditionally: (even if the log level is warn, the escape is performed).

Same in two log lines below.

Reply:
Not changing as discussed, the impact is limited in this case.